docs/advanced/handover.md

```diff
@@ -10,15 +10,15 @@ Botcopy provides an open endpoint for you to connect an ITSM Tool like Genesys,
 
 Here are the steps that proxy should follow:
 
-1. You receive a webhook call when Botcopy detects a specific context in dialogflow response, and when Botcopy bot is paused and user writes a message.
-2. You can interact with the chat user with a `message`, `channel_update`, and `agent_typing`.
+1. You receive a webhook call when Botcopy detects the context/session parameter of `bcHumanHandover`, your bot is paused and the end-user sends a message.
+2. You can interact with the end-user through `message`, `channel_update`, and `agent_typing`.
 3. You will handle the socket towards the chosen ITSM tool.
 
 ## Setup in the Botcopy Portal
 
-The handover integration is set for each individual Botcopy bot on its Connect page in the Portal. You may set the webhook URL, set contexts which will trigger the handover, and enable the storing of message history. This is also where you generate an access token to view your message history, and set an access token so the requests can authenticate with your webhook.
+The handover integration is set for each individual Botcopy bot on its Connect page in the Portal. You may set the webhook URL, a context/session parameter of `bcHumanHandover` which will trigger the handover, and enable the storing of message history. This is also where you generate your `organization access token` to view your message history, and set a `bot access token` so the requests can authenticate with your webhook.
```

Comment:
we are force to use `bcHumanHandover` for CX bots due to a bug that was fixed specifically for this session parameter value on the chat widget.
But using `bcHumanHandover` will cause a bug with Dialogflow ES, output context should be lowercase so we could write this instead.
```suggestion
The handover integration is set for each individual Botcopy bot on its Connect page in the Portal.
You may set your Live Chat Endpoint Webhook URL
If you're using Dialogflow ES set any lowercased trigger context you want like `bc-human-handover`
If you're using Dialoflow CX set `bcHumanHandover` as your trigger session parameter
Enable the storing of message history.
Generate your `organization access token`, that's used by Botcopy to authenticate you when you make a POST request to `https://api.botcopy.com/webhooks/handover/push`
Set a `bot access token` that you will use to authenticate Botcopy when Botcopy is making POST requests to your Live Chat Endpoint Webhook.
```